packages/ember-metal/lib/property_get.js

```diff
@@ -51,6 +51,8 @@ export let UNHANDLED_GET = symbol('UNHANDLED_GET');
   @public
 */
 export function get(obj, keyName) {
+  Ember.assert(`Get must be called with two arguments; an object and a property key`, arguments.length >= 2);
+
```

Comment:
It seems that `_getPath` ([here](https://github.com/martndemus/ember.js/blob/cleanup-ember-get/packages/ember-metal/lib/property_get.js#L174)), calls get with three arguments, can anyone explain me why?